web.js

```diff
@@ -354,7 +354,29 @@ module.exports = function( options ) {
       tx$:  seneca.root.idgen()
     })
 
-    next_service(req,res,next,0)
+    next_service(req,res,function(err, response){
```

Comment:
this function is a wrapper around next to handle possible errors - it should be explicit rather than inline, so give it a name - e.g. `next_service(req,res,error_handler,0)`